src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -128,13 +131,15 @@ $3dTiles_Provider.prototype.executeCommand = function executeCommand(command) {
         obj.layers.set(layer.threejsLayer);
     };
     if (urlSuffix) {
-        const url = metadata.urlPrefix + urlSuffix;
+        let url = metadata.urlPrefix + urlSuffix;
```

Comment:
Is this code trying to find out if `urlSuffix` is a relative or absolute url?

Maybe worth adding a comment specifying why you're doing this.